tests/post_training/test_quantize_conformance.py

```diff
@@ -85,8 +85,10 @@ def fixture_wc_reference_data():
         data = yaml.safe_load(f)
         fp32_test_cases = defaultdict(dict)
         for test_case_name in data:
+            data[test_case_name]["atol"] = 1e-5
```

Comment:
```suggestion
            data[test_case_name]["atol"] = data[test_case_name].get("atol", 1e-5)
```
To have option to change from json
and do same for fp32 in 90 and 91 lines.

Reply:
thanks, corrected